src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -257,9 +260,33 @@
     </Compile>
     <Compile Include="System\Security\Cryptography\ECCng.ImportExport.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true' AND '$(TargetsWindows)' == 'true' AND ('$(TargetGroup)'=='' OR '$(TargetGroup)'=='netstandard1.7')">
```

Comment:
We're clearly going to need something better for these conditions, but that can be solved later.